azure-monitor-opentelemetry-distro/azure/monitor/opentelemetry/distro/distro.py

```diff
@@ -12,14 +12,26 @@
 )
 from opentelemetry.instrumentation.distro import BaseDistro
 
+_logger = logging.getLogger(__name__)
+
 
 class AzureMonitorDistro(BaseDistro):
-    def _configure(self, **kwargs):
-        # TODO: Uncomment when logging is out of preview
-        # environ.setdefault(OTEL_LOGS_EXPORTER, "azure_monitor_opentelemetry_exporter")
-        environ.setdefault(
-            OTEL_METRICS_EXPORTER, "azure_monitor_opentelemetry_exporter"
-        )
-        environ.setdefault(
-            OTEL_TRACES_EXPORTER, "azure_monitor_opentelemetry_exporter"
-        )
```

Comment:
Since configure_opentelemetry is separate, are you suggesting we evenetually incorporate both into distro.configure but have it default to auto?

Reply:
No, autoinstrumentation and manual will have separate logic pipelines. Manual instrumentation is strictly done with `configure_opentelemetry` and autoinstrumentation with opentelemetry `BaseDistro` mechanism.